Source/WebCore/platform/graphics/skia/GraphicsContextSkia.cpp

```diff
@@ -289,7 +292,9 @@ void GraphicsContextSkia::drawLine(const FloatPoint& point1, const FloatPoint& p
     if (!makeGLContextCurrentIfNeeded())
         return;
 
-    canvas().drawLine(SkFloatToScalar(point1.x()), SkFloatToScalar(point1.y()), SkFloatToScalar(point2.x()), SkFloatToScalar(point2.y()), createFillPaint());
+    SkPaint paint;
+    setFillSource(paint);
```

Comment:
I think only the color makes sense here. Also even if it doesn't solve the problem of dashed borders, we can change the fill to stroke here, since it's clear we want a stroke.